src/documentation/ntfs_to_netex_france_specs.md

```diff
@@ -160,7 +161,7 @@ As stated in `NF_Profil NeTEx éléments communs(F) - v2.1.pdf` in chapter 5.10:
 - `true` if all `AccessibilityLimitation` are set to `true`
 - `false` if all `AccessibilityLimitation` are set to `false`
 - `partial` if some `AccessibilityLimitation` are set to `true`
-- `unknow` in other cases
+- `unknown` in other cases
```

Comment:
Just a few line above, line [160](https://github.com/CanalTP/transit_model/pull/563/files#diff-b4feb4e1eed4b42bb47fc643375c5766R160) (but I can't comment on it because it's too far from a modification), the word `sould` has been misspelled. Can you fix it?

Reply:
Done in 3218f5a155364b1bb4924be899e08b82caa2cb27